common/src/main/resources/assets/configured/lang/sv_se.json

```diff
@@ -0,0 +1,73 @@
+{
+    "configured.gui.search": "Sök...",
+    "configured.gui.reset_all": "Återställ allt",
+    "configured.gui.edit": "Redigera",
+    "configured.gui.edit_value": "Redigera värde",
+    "configured.gui.add_value": "Lägg till...",
+    "configured.gui.reset": "Återställ",
+    "configured.gui.remove": "Ta bort",
+    "configured.gui.modify": "Ändra",
+    "configured.gui.restore": "Återställ",
+    "configured.gui.select": "Välj",
+    "configured.gui.change": "Ändra",
+    "configured.gui.request": "Begär",
+    "configured.gui.save": "Spara",
+    "configured.gui.view": "Visa",
+    "configured.gui.close": "Stäng",
+    "configured.gui.apply": "Verkställ",
+    "configured.gui.home": "Hem",
+    "configured.gui.select_world": "Välj värld",
+    "configured.gui.deep_search": "Djupsökning",
+    "configured.gui.no_permission": "Du har inte behörighet att göra detta",
+    "configured.gui.not_installed": "Configured har inte installerats på denna server",
+    "configured.gui.edit_world_config": "Välj värld att redigera konfigurationen för %s",
+    "configured.gui.info": "Denna meny genererades dynamiskt av Configured. Klicka här för att öppna CurseForges sida för denna mod!",
+    "configured.gui.server_config_info": "Serverbaserade konfigurationer är unika för varje värld. Detta innebär att alla ändringar som görs i konfigurationen kommer endast att tillämpas på den valda världen.",
+    "configured.gui.restore_message": "Är du säker på att du vill återställa alla värden? Om du spelar med ett modpaket kan denna konfiguration ha modifierats av skaparen och kommer förstöra paketets avsedda upplevelse om det återställs. §6Denna handling kan inte ångras!",
+    "configured.gui.unsaved_changes": "Det finns osparade ändringar. Är du säker på att du vill sluta redigera?",
+    "configured.gui.server_config_main_menu": "Serverkonfigurationer kan endast redigeras från huvudmenyn",
+    "configured.gui.only_editable_directly": "This config can only be modified by editing the raw file on the server",
```

Comment:
Hey @Regnander, as a fellow Serbian translator I came across this PR. On a quick look I noticed that you missed to translate few labels:
- `configured.gui.only_editable_directly`
- `configured.gui.no_editing_published_lan_server`
- `configured.gui.request.invalid_config_type`
- `key.categories.configured`

If I misunderstood something, please ignore my comment.
I wish you all the best and good luck with future translations 🚀